.github/workflows/build.yml

```diff
@@ -50,26 +55,33 @@ jobs:
 
       - name: Test
         env:
-          CI: 1
-        run: pytest --skipcdf
+          SETTINGS__COGNITE__CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+          SETTINGS__COGNITE__CLIENT_ID: ${{ vars.CLIENT_ID }}
+          SETTINGS__COGNITE__PROJECT: ${{ vars.PROJECT }}
+          SETTINGS__COGNITE__CDF_CLUSTER: ${{ vars.CLUSTER }}
+          SETTINGS__COGNITE__TENANT_ID: ${{ vars.TENANT_ID }}
```

Comment:
**question**: Is this pointing to [environment variables in the CI environment](https://github.com/cognitedata/power-ops-sdk/settings/environments/1542063228/edit#:~:text=10%20months%20ago-,Environment%20variables,-Variables%20are%20used)? I don't see them there.

Reply:
They're on the repo level because all the envs need them